index.ts

```diff
@@ -56,10 +56,13 @@ collect.set('isCompare', [
 
 export const isDashboard = (url: URL | HTMLAnchorElement | Location = location): boolean => !isGist(url) && /^$|^(orgs\/[^/]+\/)?dashboard(\/|$)/.test(getCleanPathname(url));
 collect.set('isDashboard', [
+	'https://github.com///',
```

Comment:
As it stands two trailing slashes here is OK without this patch so an extra test case with three is needed.